javascript/index.js

```diff
@@ -14,52 +15,122 @@ let milUni = document.getElementById('milUni');
 let splits = document.getElementById('splits');
 
 function printTime() {
-  // ... your code goes here
+  getMinutes(); //return minutes
+  getSeconds()// return seconds
+  twoDigitsNumber(minutes)
```

Comment:
the minutes and seconds variables do not exist in the function. If you want to use the value of the getMinutes and getFunction you need to save those in the variables.